src/sortStudents.ts

```diff
@@ -1,16 +1,57 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Student[], sortBy: SortType, order: SortOrder,
+): Student[] {
+  const studArr = [...students];
+
+  const sorter = studArr.sort((a: Student, b: Student) => {
```

Comment:
The sort method is mutable, so there is no need to create a variable for a sorted array
```suggestion
  studArr.sort((a: Student, b: Student) => {
```